neuland/digitizing/R3BNeulandHitMon.h

```diff
@@ -35,55 +37,46 @@ class TH1I;
 class R3BNeulandHitMon : public FairTask
 {
   public:
-    explicit R3BNeulandHitMon(TString input = "NeulandHits",
-                              TString output = "NeulandHitMon",
-                              const Option_t* option = "");
-
-    ~R3BNeulandHitMon() override = default;
-
-    // No copy and no move is allowed (Rule of three/five)
-    R3BNeulandHitMon(const R3BNeulandHitMon&) = delete;            // copy constructor
-    R3BNeulandHitMon(R3BNeulandHitMon&&) = delete;                 // move constructor
-    R3BNeulandHitMon& operator=(const R3BNeulandHitMon&) = delete; // copy assignment
-    R3BNeulandHitMon& operator=(R3BNeulandHitMon&&) = delete;      // move assignment
+    explicit R3BNeulandHitMon(const Option_t* option = "");
 
   protected:
-    InitStatus Init() override;
+    auto Init() -> InitStatus override;
     void Finish() override;
 
   public:
     void Exec(Option_t* /*option*/) override;
 
-    void SetDistanceToTarget(double distance) { fDistanceToTarget = distance; }
+    void SetDistanceToTarget(double distance) { distance_to_target_ = distance; }
 
   private:
-    TString fOutput;
-
-    TCAInputConnector<R3BNeulandHit> fHits;
-
-    double fDistanceToTarget = 0.;
-
-    Bool_t fIs3DTrackEnabled = false;
-    TH3D* fh3 = nullptr;
-
-    TH1D* hTime = nullptr;
-    TH1D* hTimeAdj = nullptr;
-    TH1I* hMult = nullptr;
-    TH1D* hDepth = nullptr;
-    TH1D* hForemostEnergy = nullptr;
-    TH1D* hSternmostEnergy = nullptr;
-    TH2D* hDepthVSForemostEnergy = nullptr;
-    TH2D* hDepthVSSternmostEnergy = nullptr;
-    TH1D* hEtot = nullptr;
-    TH1D* hE = nullptr;
-    TH1D* hX = nullptr;
-    TH1D* hY = nullptr;
+    std::string output_;
```

Comment:
Prefer initializing member variables here, instead of in the constructor.

```suggestion
    std::string output_ {"NeulandHitMon"};
```